errbot/cli.py

```diff
@@ -131,6 +131,8 @@ def main():
                                 help='force local text backend')
     mode_selection.add_argument('-G', '--graphic', dest="backend", action='store_const', const="Graphic",
                                 help='force local graphical backend')
+    mode_selection.add_argument('-D', '--dark-text', dest="dark_text", action='store_true',
+                                help='force the log output to be dark, for use with light backgrounds')
```

Comment:
I think 9 out of 10 people won't use this option, so while I have no objections against `--dark-text` at all, I'd prefer to keep `-D` free, at least for the time being.